### ISSUE TITLE
Reopen unit test in CI

### ISSUE THREAD

**Issue by seeflood** [mosn/layotto#529]
Our CI won't run unit tests due to #483 . Now it's time to reopen it.

We shall:
- [ ] Reopen unit test in CI
https://github.com/mosn/layotto/blob/223945e4a49c499ac66a5c13b267af6b080a79a1/.github/workflows/layotto-ci.yml#L59

- [ ] Don't test code under `./cmd/...` package
https://github.com/mosn/layotto/blob/223945e4a49c499ac66a5c13b267af6b080a79a1/make/golang.mk#L87
- [ ] Test code under `sdk/go-sdk` package
- [ ] Test code under `component` package after #528 fixed

**Comment by seeflood** [comment]
@Xunzhuo Hi, can I assign it to you?

**Comment by Xunzhuo** [comment]
Sure, this is what I want too, since the new pipelines are running, there are a lot of things need to do make it stable.

**Comment by Xunzhuo** [comment]
I will start a new issue to cover all the tasks related to the new Pipelines, make it stable.

**Comment by seeflood** [comment]
ok!

**Comment by Xunzhuo** [comment]
> Our CI won't run unit tests due to #483 . Now it's time to reopen it.
> 
> We shall:
> 
> * [ ]  Reopen unit test in CI
>   https://github.com/mosn/layotto/blob/223945e4a49c499ac66a5c13b267af6b080a79a1/.github/workflows/layotto-ci.yml#L59
> * [ ]  Don't test code under `./cmd/...` package
>   https://github.com/mosn/layotto/blob/223945e4a49c499ac66a5c13b267af6b080a79a1/make/golang.mk#L87
> * [ ]  Test code under `sdk/go-sdk` package
> * [ ]  Test code under `component` package after [Unit tests fail in `component` package #528](https://github.com/mosn/layotto/issues/528) fixed

Should not we test cases in `pkg` ?

**Comment by Xunzhuo** [comment]
cc @seeflood, please check this if it is missed:)

**Comment by seeflood** [comment]
>Should not we test cases in pkg ?

@Xunzhuo We should. I didn't put it here because your scripts already test this package
